onadata/libs/permissions.py

```diff
@@ -300,47 +347,57 @@ class OwnerRole(Role):
 # Memoize a class to permissions dict.
 for role in ROLES.values():
     role.class_to_permissions = defaultdict(list)
-    [role.class_to_permissions[k].append(p) for p, k in role.permissions]
+    for p, k in role.permissions:
+        role.class_to_permissions[k].append(p)
 
 
 def is_organization(obj):
-    """Some OrganizationProfiles have a pointer to the UserProfile, but no
+    """
+    Some OrganizationProfiles have a pointer to the UserProfile, but no
     UserProfiles do. Check for that first since it avoids a database hit.
     """
     try:
-        hasattr(obj, 'userprofile_ptr') or obj.organizationprofile
-        return True
+        return (hasattr(obj, 'userprofile_ptr') or
+                obj.organizationprofile is not None)
     except OrganizationProfile.DoesNotExist:
         return False
 
 
 def get_role(permissions, obj):
-    for role in reversed(ROLES_ORDERED):
-        if role.has_role(permissions, obj):
-            return role.name
+    """
+    Return the user role for the given obj permissions.
+    """
+    for _role in reversed(ROLES_ORDERED):
```

Comment:
why change? this is not a reserved word, are you concerned it'll conflict/confuse with the class name? The case is different, but I suppose that's fair

Reply:
Pylint, it actually has to do with the code with the list comprehension.